### DISCUSSION TITLE
ModuleNotFoundError: No module names 'mgis'

### DISCUSSION THREAD

**Discussion by Hirshikesh** [thelfer/tfel#461] [General]
I have followed all the steps mentioned in section 7: A Quick installation procedure on Ubuntu (https://tfel.sourceforge.net/install.html). However, when I try to import mgis.fenics it says:

 ModuleNotFoundError: No module names 'mgis'

Is there anything I am Missing?

**Comment by thelfer** [comment]
Hi @Hirshikesh,
You may want to have a look at [this discussion](https://github.com/thelfer/MFrontGenericInterfaceSupport/issues/127) for a very similar question.

**Reply by Hirshikesh** [reply]
I have tried to use script install.sh, but shows error. See the attached screenshot
![Screenshot 2023-12-14 094729](https://github.com/thelfer/tfel/assets/32254574/78801d3d-7ed0-4447-9ced-7692e524b50e)
.

**Reply by thelfer** [reply]
Did you read the first lines of the scripts ? It gives the package to install. Be aware that you probably don't need tfel-plot and tfel-editor, so you can remove everything related to qt and the correspondind lines
